manager/src/main/java/org/openremote/manager/web/ManagerWebService.java

```diff
@@ -77,7 +77,7 @@ protected interface ServerVariableMixin {
     public static final String OR_APP_DOCROOT = "OR_APP_DOCROOT";
     public static final String OR_APP_DOCROOT_DEFAULT = "manager/src/web";
     public static final String OR_CUSTOM_APP_DOCROOT = "OR_CUSTOM_APP_DOCROOT";
-    public static final String OR_CUSTOM_APP_DOCROOT_DEFAULT = "ui/app";
+    public static final String OR_CUSTOM_APP_DOCROOT_DEFAULT = "deployment/manager/app";
```

Comment:
(https://github.com/openremote/openremote/blob/master/profile/deploy.yml#L228) Changed it so it has the same value as the value that is used within docker.

Reply:
Ok, I will change it back. Is there another variable that I can use? so that it also works in development? If not I will go and hardcode a path. It's just not my preference